package-overrides/npm/redbox-react@1.1.1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "registry": "npm",
+  "main": "dist/redbox",
```

Comment:
@timbur because most projects out there don't support SystemJS and jspm, the project does require more time investment to make things work. We've done a lot to try and support existing things, but there will always be edge cases that make it a bumpy ride. Checking in overrides, posting issues and project maintainers checking their projects against jspm all contribute to improving this, which will take some time to match the development experience in other systems.